README.md

```diff
@@ -31,43 +31,111 @@ To do this, you must copy the following files in the init directory :
+| cgmes-boundary-import-job     |        |    🗸    |       |        |  🗸    |        |        |         |
+| cgmes-boundary-server         |        |    🗸    |       |        |  🗸    |        |        |         |
+| merge-notification-server     |        |    🗸    |       |        |  🗸    |        |        |         |
+| merge-orchestrator-server     |        |    🗸    |       |        |  🗸    |        |        |         |
+| gridstudy-app                 |        |         |   🗸   |        |  🗸    |        |        |         |
+| cgmes-gl-server               |        |         |   🗸   |        |  🗸    |        |        |         |
+| directory-notification-server |        |         |   🗸   |        |  🗸    |        |        |         |
+| directory-server              |        |         |   🗸   |        |  🗸    |        |        |         |
+| dynamic-simulation-server     |        |         |   🗸   |        |  🗸    |        |        |         |
+| explore-server                |        |         |   🗸   |        |  🗸    |        |        |         |
+| geo-data-server               |        |         |   🗸   |        |  🗸    |        |        |         |
+| gridexplore-app               |        |         |   🗸   |        |  🗸    |        |        |         |
+| network-map-server            |        |         |   🗸   |        |  🗸    |        |        |         |
+| network-modification-server   |        |         |   🗸   |        |  🗸    |        |        |         |
+| odre-server                   |        |         |   🗸   |        |  🗸    |        |        |         |
+| security-analysis-server      |        |         |   🗸   |        |  🗸    |        |        |         |
+| sensitivity-analysis-server   |        |         |   🗸   |        |  🗸    |        |        |         |
+| shortcircuit-server           |        |         |   🗸   |        |  🗸    |        |        |         |
+| single-line-diagram-server    |        |         |   🗸   |        |  🗸    |        |        |         |
+| study-notification-server     |        |         |   🗸   |        |  🗸    |        |        |         |
+| study-server                  |        |         |   🗸   |        |  🗸    |        |        |         |
+| timeseries-server             |        |         |   🗸   |        |  🗸    |        |        |         |
+| voltage-init-server           |        |         |   🗸   |        |  🗸    |        |        |         |
+| case-import-server            |        |         |       |        |       |   🗸    |        |         |
+
+To use a profile, you use simply:
+```shell
+$ cd docker-compose
+$ docker compose --profile suite <cmd>
+```
+
+You can also combine multiple profiles:
+```shell
+$ docker compose --profile study --profile mapping <cmd>
+```
+But please note that services/container who belongs to at least one profile can't be accessed if the profile isn't specified.
```

Comment:
Why do you say this ? For me it works. 
It's just that you need to be careful for the dependencies according to the doc (`depends_on`).
I missed something ?

Reply:
Actually `docker compose up` start everything but `down` does not work well if you don't specify the `all` profile...
But `docker compose stop` still works well.

Reply:
Hum, maybe I wasn't clear in the doc.  
What I want to say is if you do for example `docker compose stop study-server` or `docker compose --profile merging stop study-server` it would not work because study-server is in the `study` and `suite` profiles, so not visible until one of the profiles it is attached (`study` or `suite` ... or `all`) is passed in the cli.

Reply:
I tried the first one and it works for me.

Reply:
After some tests, the [auto-starting-profiles](https://docs.docker.com/compose/profiles/#auto-starting-profiles-and-dependency-resolution) documentation seems to be outdated
  * `up` and `down` **need** profiles explicitly
  * `start`/`stop`/`restart` _don't care_ of profiles, they only need the container(s) to be created by a previously `up`
    * and check `depends_on`

➡️ Need to update the readme